scripts/kpca.py

```diff
@@ -10,23 +10,26 @@
 from asaplib.pca import KernelPCA
 from asaplib.plot import *
 
+from sklearn.decomposition import KernelPCA as kpca_sklearn
 
-def main(fmat, fxyz, ftags, fcolor, colorscol, prefix, output, kpca_d, pc1, pc2, adtext):
+
+def main(fmat, fxyz, ftags, fcolor, colorscol, prefix, output, kpca_d, pc1, pc2, adtext, use_sklearn):
     """
 
     Parameters
     ----------
-    fmat
-    fxyz
-    ftags
-    fcolor
-    colorscol
-    prefix
-    output
+    fmat: Location of descriptor matrix file or name of the tags in ase xyz file. You can use gen_descriptors.py to compute it.
+    fxyz: Location of xyz file for reading the properties.
+    ftags: Location of tags for the first M samples. Plot the tags on the umap.
+    fcolor: Location of a file or name of the tags in ase xyz file. It should contain properties for all samples (N floats) used to color the scatterplot'
+    colorscol: The column number of the properties used for the coloring. Starts from 0.
+    prefix: Filename prefix, default is ASAP
+    output: The format for output files ([xyz], [matrix]). Default is xyz.
     kpca_d: number of dimensions
-    pc1
-    pc2
-    adtext
+    pc1: Plot the projection along which principle axes
+    pc2: Plot the projection along which principle axes
+    adtext: Whether to adjust the texts (True/False)
+    use_morgan: Whether to use Morgan fingerprints. True for the photoswitch example.
```

Comment:
New flag. Default is False. True only for photoswitch example.

Reply:
In the redesign of the code, this will be removed. But for the moment it's okay

Reply:
Do you want to merge the redesign first and then I can alter this example PR to fit with the redesign?

Reply:
That will be great. Okay, so I'll merge the redesign soon (in the next couple of days).

Please commit the new examples (*xyz and other data files for the photoswtich example first). Please add some texts/draft fingers about this example to the overleaf doc too. Thx.